inscriptions/ISic3094.xml

```diff
@@ -177,11 +182,26 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/><gap reason="lost" quantity="1" unit="line"/>
```

Comment:
Ferrua's drawing (fig. 1.d) shows one illegible character (although he proposes reading an E) in line 1:
![image](https://user-images.githubusercontent.com/25080386/104302624-eeb18f00-54c0-11eb-936a-b6b1804d04e3.png)